snapshot/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM alpine:latest
+COPY data/ /
```

Comment:
This fails because there's no `data/` directory.  Maybe add and commit `data/.keep`?

Reply:
there aren't several subdirectories?
```
birkland@MSEL-DRCC07:~/projects/test/idc-dev/local/snapshot/data$ ls
drupal  mariadb-data  mariadb-files  matomo-config  minio  solr
```

What do you see, and what is the error?

Reply:
oh, I must not have comitted them!